docs/index.html

```diff
@@ -46,7 +46,7 @@
de></p><div class="sample-container"><pre><code class="block lang-kotlin" theme="idea">val builder = ATThunderSDK.createBuilder()<br>    .setAdTonosKey(&quot;XXXXX&quot;)          // Sets developer key.<br>    .setLanguage(&quot;en&quot;)               // Sets user language, if different than a system defined.<br>    .setAdType(VastAdType.bannerAd)  // Sets the type of ad. By default VastAdType.regular is set.<br>    .build()</code></pre><span class="top-right-position"><span class="copy-icon"></span><div class="copy-popup-wrapper popup-to-left"><span class="copy-popup-icon"></span><span>Content copied to clipboard</span></div></span></div><h3 class=""> Banner view setup</h3><p class="paragraph">It's necessary to set banner container view declared in xml if you want to use banner ads. It's also necessary to add banner view to all activity layouts if banner should be visible across all activities. Additionally to avoid lifecycle problems add setBannerContainer and addLifecycleObserver methods in onResume of all activities.</p><div class="sample-container"><pre><code class="block lang-xml" theme="idea">&lt;com.adtonos.sandstormsdk.core.SandstormBannerView<br>    android:id=&quot;@+id/sandstormBannerView&quot;<br>    android:layout_width=&quot;wrap_content&quot;<br>    android:layout_height=&quot;wrap_content&quot; /&gt;</code></pre><span class="top-right-position"><span class="copy-icon"></span><div class="copy-popup-wrapper popup-to-left"><span class="copy-popup-icon"></span><span>Content copied to clipboard</span></div></span></div><div class="sample-container"><pre><code class="block lang-kotlin" theme="idea">ATSandstormSDK.setBannerContainer(findViewById(R.id.sandstormBannerView))<br>ATSandstormSDK.addLifecycleObserver(lifecycle)</code></pre><span class="top-right-position"><span class="copy-icon"></span><div class="copy-popup-wrapper popup-to-left"><span class="copy-popup-icon"></span><span>Content copied to clipboard</span></div></span></div><h3 class=""> Setting the banner position</h3><p class="paragraph">The SDK offers to set the banner position, to do this use the following method:</p><div class="sample-container"><pre><code class="block lang-kotlin" theme="idea">ATSandstormSDK.setBannerPosition(SandstormBannerPosition.bottom) </code></pre><span class="top-right-position"><span class="copy-icon"></span><div class="copy-popup-wrapper popup-to-left"><span class="copy-popup-icon"></span><span>Content copied to clipboard</span></div></span></div><p class="paragraph">It is possible to display the banner view at the <strong>top</strong> of the screen or at the <strong>bottom</strong>.</p><p class="paragraph">By default, the banner will be displayed at the <strong>top</strong> of the screen. The method needs to be called before the ads are played.</p><h2 class=""> Documentation</h2><p class="paragraph">Please see extended documentation from repository to get information about possible errors and other methods that can be invoked.</p></div></div>
```

Comment:
These `-> TODO()` calls in the pattern-matched when() are supposed to be there, right?